From f6d4f709ec57835b3fa4a660239a07321c9d02ff Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sun, 14 Feb 2016 08:49:09 +0100
Subject: [PATCH] Fix flags for KFontDialog::getFont()

Pass a proper value for flags, instead of a boolean.

This fixes the build with GCC 6.
---
 src/dialogs/setupdisplay.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dialogs/setupdisplay.cpp b/src/dialogs/setupdisplay.cpp
index 43635e8..eb05874 100644
--- a/src/dialogs/setupdisplay.cpp
+++ b/src/dialogs/setupdisplay.cpp
@@ -562,7 +562,7 @@ void SetupDisplay::setShown( int id )
 void SetupDisplay::setFont()
 {
 	KreDisplayItem *item = *node_item_map->find( m_currNodeId );
-	if ( KFontDialog::getFont( item->font, false, view() ) == QDialog::Accepted ) {
+	if ( KFontDialog::getFont( item->font, KFontChooser::NoDisplayFlags, view() ) == QDialog::Accepted ) {
 		m_currentItem = item;
 		loadFont(m_currNodeId,item->font);
 		m_currentItem = 0;
-- 
2.7.0

